protostar/commands/call_command.py

```diff
@@ -1,24 +1,44 @@
 import json
+from dataclasses import dataclass
 from logging import Logger
-from typing import Any, Optional, Dict
+from typing import Any, Optional
 
 from starknet_py.net.gateway_client import GatewayClient
 
 from protostar.cli import (
     NetworkCommandUtil,
+    OutputCommandUtil,
     ProtostarArgument,
     ProtostarCommand,
 )
+from protostar.io import LogColorProvider, StructuredMessage
 from protostar.starknet_gateway import GatewayFacadeFactory
 
 
+@dataclass
+class SuccessfulCallMessage(StructuredMessage):
+    response: Any
+
+    def format_human(self, fmt: LogColorProvider) -> str:
+        return f"""\
+{fmt.colorize("GREEN", "Call successful.")}
+Response:
+{json.dumps(self.response._asdict(), indent=4)}\
+"""
+
+    def format_object(self) -> dict:
```

Comment:
```suggestion
    def as_dict(self) -> dict:
```
maybe?

Reply:
https://github.com/software-mansion/protostar/pull/1137#discussion_r1018963715

Reply:
`format_dict` maybe? It returns dict not an object

Reply:
done